code/workspaces/web-app/src/pages/PageTemplate.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Typography, withStyles } from '@material-ui/core';
+import Footer from '../components/app/Footer';
+
+const style = theme => ({
+  pageTemplate: {
+    display: 'flex',
+    flexDirection: 'column',
+    minHeight: '100%',
+  },
+  contentArea: {
+    flexGrow: 1,
+  },
+});
+
+function PageTemplate({ title, children, classes }) {
```

Comment:
I like what you've done here, but is `PageTemplate` the right name? Could it just be `Page`? Might look better in the usage then?